kotlin/README.md

```diff
@@ -0,0 +1,7 @@
+# Apache Fury™ Kotlin
+
+Provides Fury support for Kotlin Serialization on JVM:
```

Comment:
Could we clear that Fury has already support most kotlin types well, it's easy for users to think that fury only support those types